tests/php/Model/BrokenExternalLinkTest.php

```diff
@@ -32,4 +32,34 @@ public function httpCodeProvider()
             [789, '789 (Unknown Response Code)'],
         ];
     }
+
+    /**
+     * @return array[]
+     */
+    public function permissionProvider()
```

Comment:
```suggestion
    public function permissionProvider(): array
```
Strongly type the method instead of using phpdocs.